app/code/Magento/Customer/Model/AttributeMetadataResolver.php

```diff
@@ -75,25 +78,34 @@ class AttributeMetadataResolver
      */
     private $shareConfig;
 
+    /**
+     * @var PrefixSuffixWithWebsites
+     */
+    private $prefixSuffixWithWebsites;
+
     /**
      * @param CountryWithWebsites $countryWithWebsiteSource
      * @param EavValidationRules $eavValidationRules
      * @param FileUploaderDataResolver $fileUploaderDataResolver
      * @param ContextInterface $context
      * @param ShareConfig $shareConfig
+     * @param PrefixSuffixWithWebsites|null $prefixSuffixWithWebsites
      */
     public function __construct(
         CountryWithWebsites $countryWithWebsiteSource,
         EavValidationRules $eavValidationRules,
         FileUploaderDataResolver $fileUploaderDataResolver,
         ContextInterface $context,
-        ShareConfig $shareConfig
+        ShareConfig $shareConfig,
+        ?PrefixSuffixWithWebsites $prefixSuffixWithWebsites = null
     ) {
         $this->countryWithWebsiteSource = $countryWithWebsiteSource;
         $this->eavValidationRules = $eavValidationRules;
         $this->fileUploaderDataResolver = $fileUploaderDataResolver;
         $this->context = $context;
         $this->shareConfig = $shareConfig;
+        $this->prefixSuffixWithWebsites = $prefixSuffixWithWebsites ?: ObjectManager::getInstance()
```

Comment:
For 2.4.0 you can inject dependencies naturally ;)

Reply:
Done @slavvka :)